docs/getting_started.rst

```diff
@@ -335,7 +335,7 @@ To ensure a more efficient workflow, install appropriate editor plugins for Vue.
 Sample resources and data
 -------------------------
 
-Once you have the server running, proceed to import some channels and resources. To quickly import all available and supported Kolibri resource types, use the token ``nakav-mafak`` for the `Kolibri QA channel <https://kolibri-beta.learningequality.org/en/learn/#/topics/95a52b386f2c485cb97dd60901674a98>`__ (~350MB).
+Once you have the server running, proceed to import some channels and resources. To quickly import all available and supported Kolibri resource types, `import with the token <https://kolibri.readthedocs.io/en/latest/manage/resources.html?highlight=import#import-with-token>`__  ``nakav-mafak`` for the `Kolibri QA channel <https://kolibri-beta.learningequality.org/en/learn/#/topics/95a52b386f2c485cb97dd60901674a98>`__ (~350MB).
```

Comment:
This was pre-existing, but `kolibri-beta` doesn't exist any more! I think just updating this to `kolibri-demo` should be fine, although BCK is currently having some difficulties, so I can't be sure!

Reply:
We can update this in follow up, so as not to hold this up further!